evals/scripts/get_premium_fans_generator.py

```diff
@@ -3,30 +3,9 @@
 import string
 from get_context import get_context
 from get_instruction import get_instruction
+from create_yaml_data import create_yaml, create_data
 
-CURRENT_DIR = os.path.dirname(__file__)
-REGISTRY_PATH = os.path.join(CURRENT_DIR, "../registry")
-
-YAML = """
-get_premium_fans:
-  id: get_premium_fans.dev.v0
-  description: Evaluates the model's ability to analyze music streaming and fan engagement data, including metrics like premium vs free subscribers, playlist engagement, artist performance, and listener behavior across different streaming platforms.
-  metrics: [accuracy]
-get_premium_fans.dev.v0:
-  class: evals.elsuite.modelgraded.classify:ModelBasedClassify
-  args:
-    samples_jsonl: get_premium_fans/samples.jsonl
-    eval_type: cot_classify
-    modelgraded_spec: closedqa
-    modelgraded_spec_args:
-      criteria: "correctness: The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
-""".strip()
-
-yaml_file = os.path.join(REGISTRY_PATH, "evals/get_premium_fans.yaml")
-with open(yaml_file, "w") as yf:
-    yf.write(YAML)
-    
-data_dir = os.path.join(REGISTRY_PATH, "data/get_premium_fans")
+create_yaml("get_premium_fans")   
 
 mock_data = [
```

Comment:
mock_data is the same in all files too, right?
- Could this also be DRY for easier `extension` and `maintainability`?